### ISSUE TITLE
changing target

### ISSUE THREAD

**Issue by lewandowskijake1123** [radical-cybertools/radical.entk#228]
When changing the target and my username on the target machine(bridges in this case), I included the line " 'username': 'jel203'" in my code. The API guide says that username is an attribute. My username seems to be ignored and resouce_manager.py file does not seem to include username as a variable. Am I missing something here?

Also, wfp_terminate does not seem to be an object either, which I get as an error as well.

**Comment by vivek-bala** [comment]
You are correct. The documentation/example needs to be updated with the following details. Thanks for catching that. The username attribute is removed from the components. Instead you are required to create an ssh config file:

```
vi $HOME/.ssh/config
```
Add the following (note the tabs):
```
Host <resource label>
    Hostname <resource url>
    User <username>
```
example:
```
$ cat .ssh/config 
Host xsede.comet comet.sdsc.xsede.org
    Hostname comet.sdsc.xsede.org
    User vivek

Host xsede.stampede stampede.tacc.utexas.edu stampede.tacc.xsede.org
    Hostname stampede.tacc.utexas.edu
    User vivek

Host xsede.supermic supermic.cct-lsu.xsede.org
    Hostname supermic.cct-lsu.xsede.org
    User vivek
```

**Comment by vivek-bala** [comment]
This has been resolved in 0.7.4.
